webpack.config.storybook.js

```diff
@@ -4,7 +4,7 @@ const wixStorybookConfig = require('yoshi/config/webpack.config.storybook');
 const StylableWebpackPlugin = require('@stylable/webpack-plugin');
 const project = require('yoshi-config');
 const { resolveNamespaceFactory } = require('@stylable/node');
-const autoprefixer = require('autoprefixer')({ grid: true, browsers: ['>1%'] });
+const autoprefixer = require('autoprefixer')({ grid: true, overrideBrowserslist: ['>1%'] });
```

Comment:
what is the future of this configuration?
it seems to me like we need to understand if it's related only to storybook or also to actual build.
As I see it - this won't work on production builds unless done in `yoshi`

Reply:
Right, we should check with `yoshi` guys.
At least it's "only" for ie11 support